### PR TITLE
add gin component to app

### PR DESCRIPTION
**Pull request by kendallm360**
## Summary:
Added gin component to the app

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [x] New feature

## Where are the Changes:
GinDeck folder
GinContainer folder
App.tsx
Specialtycontainer.tsx

## How was this tested:
Chrome/React dev tools

### CODE REVIEW

File: src/components/GinContainer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import ginImage from "../../images/ginSour.jpeg";
+import { Link } from "react-router-dom";
+
+const GinContainer: FC = () => {
+  return (
+    <div>
+      <Link to="/GinDeck">
+        <div className="ginButton">
+          <img className="ginImage" src={ginImage} />
+          <h3>20 Gins that will get you loose</h3>
```
Comment (jordan-sullivan) [inline src/components/GinContainer/index.tsx:12]: Love it!

File: src/components/Types.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ export type TypeState = {
   drinkList: Drink[];
   id?: string;
   error?: false | true;
+  setDrinkList?: (active: Drink[]) => void;
```
Comment (jordan-sullivan) [inline src/components/Types.ts:5]: ahh, I see how this can be truck to navigate with Hooks and TS. We can talk about more ideas tomorrow maybe? -Jordan

**Review by jordan-sullivan — APPROVED** [review]
Great effort adding the Gin components K! # specialties really makes the app feel fuller.

**Review by nhodnett — APPROVED** [review]
Solid work! Looking really good! Nice choice of image...

**Pull request merged**